bin/cron/update_dotd

```diff
@@ -24,7 +24,11 @@ def main
 
   # Set the new developers room topic
   new_topic = current_topic.sub(/^.+?;/, "DOTD: @#{primary_dotd_user};")
-  Slack.update_topic('developers', new_topic) unless new_topic == current_topic
+  unless new_topic == current_topic
+    Slack.update_topic('developers', new_topic)
+
+    Slack.remind(Slack.user_id(DevelopersTopic.dotd), Time.now.to_i + 3600, msg)
```

Comment:
won't you need the `msg` to be defined in this file also? for verification, I would suggest running whatever is inside this block once in rails console on whichever machine runs this cronjob